src/lib/module-info.ts

```diff
@@ -83,7 +78,7 @@ export default async function getModuleInfo(
         }
     }
 
-    return { declFiles: sort(all.keys()), dependencies, declaredModules, globals: sort(globals) };
```

Comment:
I moved the `sort(all.keys())` out of this function since it didn't particularly belong.